rnaseq-cgl-pipeline/wrapper.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
+import shutil
+import subprocess
+import json
+import logging
+from uuid import uuid4
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger()
+
+
+def call_pipeline(mount, args):
+    uuid = 'Toil-RNAseq-' + str(uuid4())
+    if not os.path.exists(mount + uuid):
```

Comment:
`isdir` instead of `exists`. If it's a file, it will cause makedir to fail with an obvious error. With `exists` you kick the can down the alley.